plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -536,6 +547,26 @@ SecRule REQUEST_FILENAME "@contains /apps/files_sharing/settings/defaultAccept"
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
 
+# When trying to delete profile picture in Personal --> Personal info
+SecRule REQUEST_FILENAME "@contains /avatar" \
```

Comment:
This one looks too wide. Is it possible to make it less generic?

Reply:
This is the relevant log for that part:- 


`[Tue Apr 04 22:19:42.908426 2023] [:error] [pid 56994] [client 127.0.0.1:47434] [client 127.0.0.1]`
`ModSecurity: Warning. Match of "within %{tx.allowed_methods}" against "REQUEST_METHOD" required.`
`[file "/etc/modsecurity/coreruleset/rules/REQUEST-911-METHOD-ENFORCEMENT.conf"] [line "44"]`
`[id "911100"] [msg "Method is not allowed by policy"] [data "DELETE"] [severity "CRITICAL"] [ver`
`"OWASP_CRS/4.0.0-rc1"] [tag "application-multi"] [tag "language-multi"] [tag "platform-multi"] [tag`
`"attack-generic"] [tag "paranoia-level/1"] [tag "OWASP_CRS"] [tag "capec/1000/210/272/220/274"] [tag`
`"PCI/12.1"] [hostname "localhost"] [uri "/avatar"] [unique_id "ZCxVJmQsstiQe1UUjjQ0VwAAAAI"]`


What do you think about `"@rx ^/avatar$"`?

Reply:
Or maybe just `"@rx /avatar$"`, in case NextCloud is installed in a sub-folder.

Reply:
> Or maybe just `"@rx /avatar$"`, in case NextCloud is installed in a sub-folder.

Yes, this will be the best option BUT use `@endsWith` as it's much less expensive than regexp (the result will be the same).